signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -85,6 +85,9 @@ public class RegistrationService {
     @Value("${mosip.signup.send-challenge.captcha-required:true}")
     private boolean captchaRequired;
 
+    @Value("${mosip.signup.idrepo.idvid-postfix:}")
```

Comment:
This property defined in the signup service should be plugin agnostic. Can we rename this property as "mosip.signup.individualid.postfix"